managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from models import Actor
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+
+    def all(self):
+        cursor = self._connection.execute(
+            "SELECT * FROM actors"

```

Comment:
Why do you using hardcoded table name here "SELECT * FROM actors"  instead of using your viriable as you did in other places ?)